### PR TITLE
fix: Change `DEBUG=True`  by default to `DEBUG=False`

### PR DESCRIPTION
**Pull request by guilhem-barthes**
## Description

Replace the default for Debug. Currently by default it's ` True` , changed to ` False` . Fixes FL-1236.

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Pre-populate Skaffold deployment with substra-tools base docker image (not minimal) ([#736](https://github.com/Substra/substra-backend/pull/736))
 
+### Fixed
+
+- `settings.DEBUG` is not `True` by default.
```
Comment (oleobal) [inline CHANGELOG.md:22]:
```suggestion
- `settings.DEBUG` is now `True` by default.
```

Typo? I'm not sure

Comment (thbcmlowk) [reply CHANGELOG.md:22]: New default is `False`:
https://github.com/Substra/substra-backend/blob/fix/debug-by-default/backend/backend/settings/common.py#L26

Comment (oleobal) [reply CHANGELOG.md:22]: OK, I now realize my comment made no sense. What about this

```suggestion
- `settings.DEBUG` is no longer `True` by default.
```

**Review by SdgJlbl — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (oleobal) [reply CHANGELOG.md:22]: RIP suggestion o7

**Pull request merged**